blocks/product-featured-image-block/features/product-featured-image/default.test.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import { useFusionContext } from "fusion:context";
+
+import ProductFeaturedImage from "./default";
+
+const FEATURED_IMAGE_ASSET = {
+	type: "image",
+	url: "https://example.com/image.jpg",
+	auth: {
+		1: "secret",
+	},
+};
+
+describe("Product Featured Image", () => {
+	it("should render null if no global content", () => {
+		useFusionContext.mockImplementation(() => ({
+			globalContent: {},
+		}));
+		const { container } = render(<ProductFeaturedImage />);
+		expect(container.firstChild).toBe(null);
+	});
+	it("should render null if no global content object", () => {
+		useFusionContext.mockImplementation(() => ({}));
+		const { container } = render(<ProductFeaturedImage />);
+		expect(container.firstChild).toBe(null);
+	});
+	it("should render a featured image", () => {
+		useFusionContext.mockImplementation(() => ({
+			globalContent: {
+				schema: {
+					featuredImage: {
+						value: {
+							assets: [FEATURED_IMAGE_ASSET],
```

Comment:
not a lot of fallback logic if there's no schema of featuredImage. Possibly add test cases if we're not getting the schema we're expecting